src/extension/options-page/Welcome/getWelcomePageURL.tsx

```diff
@@ -13,6 +13,6 @@ export function getWelcomePageURL(query?: Query) {
         if (identifier) param.set('identifier', identifier.toText())
         return getUrl(`index.html#/home/?${param.toString()}`)
     } else {
-        return getUrl(`index.html#/initialize/`)
+        return getUrl(`index.html#/initialize${webpackEnv.target === 'Firefox' ? '/1d' : ''}`)
```

Comment:
```suggestion
        if (webpackEnv.target === 'Firefox') {
            return t('initialize_firefox_url')
        } else {
            return t('initialize_general_url')
        }
```

reference: #816

Reply:
lgtm

Reply:
This URL should be generated programmatically. I think it's not necessary to move URL fragments to I18N settings